db/migrate/20231018141450_add_cpu_cores_columns_to_resource_pools.rb

```diff
@@ -0,0 +1,34 @@
+class AddCpuCoresColumnsToResourcePools < ActiveRecord::Migration[6.0]
+  class ResourcePools < ActiveRecord::Base
+    include ActiveRecord::IdRegions
+    self.inheritance_column = :_type_disabled
+  end
+
+  def up
+    add_column :resource_pools, :cpu_cores_available, :float
+    add_column :resource_pools, :cpu_cores_reserve, :float
+    add_column :resource_pools, :cpu_cores_limit, :float
+
+    say_with_time('Removing data non-applicable data from IbmCloud::PowerVirtualServer resource pools') do
+      ResourcePools.in_my_region.where(:type => %w[ManageIQ::Providers::IbmCloud::PowerVirtualServers::CloudManager::ResourcePool]).update_all(
+        :cpu_shares  => nil,
+        :cpu_reserve => nil,
+        :cpu_limit   => nil
+      )
+    end
```

Comment:
I think newly added columns will get a default of `null`.
So I think this sql is not doing anything.